qiita_pet/templates/study_ajax/artifact_summary.html

```diff
@@ -1,5 +1,24 @@
 <script type="text/javascript">
   /*
+   * Function to delete an artifact
+   *
+   * @param aid The artifact to be removed
+   *
+   * This function executes an AJAX query to delete an artifact from the system
+   */
+  function delete_artifact(aid) {
+    $.post("/artifact/", {artifact_id: +aid })
```

Comment:
Needs to change based on portal changes. I think we should merge the other branch before adding new functionality, do you agree?